install/installer/pkg/components/ws-manager/configmap.go

```diff
@@ -113,6 +113,11 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil, err
 	}
 
+	installationShortNameSuffix := ""
+	if ctx.Config.Metadata.InstallationShortname != "" {
```

Comment:
@andrew-farries IMO we need to check for `!= "" && != "default", no? (cmp test cases below)